### PR TITLE
Add HapiGo 2.5.1

### PR DESCRIPTION
**Pull request by HapiGo**
Add HapiGo.app v2.5.1

HapiGo is a great tools on macOS for developers

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — COMMENTED** [review]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

**Comment by HapiGo** [comment]
> Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

Hello! thank you for your quick review, we just update our cask file and added feature zap stanza, await for your further review:-)

File: Casks/hapigo.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+cask "hapigo" do
+  version "2.5.1"
+  sha256 "6393038f16c30e00ac6f033787a261510b56f76e0bb4d32ae38fa08cd837047f"
+
+  url "https://github.com/HapiGo/HapiGo/releases/download/#{version}/HapiGo.dmg",
+      verified: "github.com/HapiGo/HapiGo/"
+  name "HapiGo"
+  desc "脑力工作者必备工具"
```
Comment (SMillerDev) [inline Casks/hapigo.rb:8]: can you also add a description in english? And optionally an extra name in what I assume to be mandarin

Comment (singularitti) [reply Casks/hapigo.rb:8]: In Chinese, this description means "Essential tools for knowledge workers", which I think violates the rules for adding a cask: "Do not use empty marketing jargon." As far as I understand, this app is similar to raycast or alfred. So I would suggest just use alfred's description: "Application launcher and productivity software", and it translates to "应用程序启动器和生产力软件".

Comment (SMillerDev) [reply Casks/hapigo.rb:8]: We can also drop the word `Essential`

Comment (singularitti) [reply Casks/hapigo.rb:8]: Yes, I would suggest using "Application launcher and productivity software"

**Pull request closed without merging**